Makefile

```diff
@@ -28,10 +28,10 @@ test_lint:
 # for more than once in Travis. Change `make test` in .travis.yml
 # to `make test_multi`.
 test_multi:
-	cd network; \
+	cd services/skipchain; \
```

Comment:
it's the `test_multi`-target which is used for whatever tests need to be done. It is normal for this target to change whenever somebody needs to run travis on one test only.